packages/react-devtools-scheduling-profiler/src/content-views/FlamechartView.js

```diff
@@ -240,7 +246,11 @@ class FlamechartStackLayerView extends View {
   /**
    * @private
    */
-  _handleMouseMove(interaction: MouseMoveInteraction) {
+  _handleMouseMove(
+    interaction: MouseMoveInteraction,
+    activeViewRef: ViewRef,
+    hoveredViewRef: ViewRef,
```

Comment:
In hindsight I could have just used a single fake-ref object for both of these to reduce the number of params passed around.